### DISCUSSION TITLE
Custom email templates (e.g. Brevo) for reset password email - possible?

### DISCUSSION THREAD

**Discussion by techwizzdom** [supabase/supabase#29992] [Questions]
In my app, I send transactional emails through Supabase Edge Functions, where I just pick a Brevo template and pass the content:

```js
 const response = await fetch(BREVO_API_URL, {
    method: "POST",
    headers: {
      "Content-Type": "application/json",
      "api-key": BREVO_API_KEY,
    },
    body: JSON.stringify(emailData),
  })
```

However, I can't figure out if it's possible to send a custom email for "password reset". In this case, a default Supabase email template is sent, and due to its simplicity, it often lands to spam.

I know I can customize emails with custom HTML, but is there a way to send this like all the other transactional emails? Or is the only option to manually write HTML?

**Comment by silentworks** [comment]
I'm not sure what you're asking here as if you use the default `auth.signUp` method and so on the emails are sent by Supabase. If you want to send using brevo from the edge function then you should use `.admin.generateLink` and get the url and other info back to put in your brevo template through their API. https://supabase.com/docs/reference/javascript/auth-admin-generatelink
